site/static/compare/script.js

```diff
@@ -363,8 +373,16 @@ app.component('test-cases-table', {
         },
     },
     template: `
-<div class="bench-table">
-<div class="category-title">{{ title }} benchmarks</div>
+<div class="bench-table" :id="id">
+<div class="category-title">
+  {{ title }} benchmarks
+  <span v-if="title == 'Primary'" title="To secondary benchmarks">
+    <a href="#secondary-benchmarks">⮯</a>
+  </span>
+  <span v-else title="To primary benchmarks">
+    <a href="#primary-benchmarks">⮭</a>
```

Comment:
I don't really love the pointer emojis, but I don't have a good suggestion otherwise...